exit.php

```diff
@@ -16,6 +16,7 @@
 
 # TO DO
 unset($_SESSION["user_id"]);
+$_SESSION["logout_msg"] = "Successful log out.";
 header("Location: index.php");
 
 $view = new View("exit");
```

Comment:
This view will not be seen by the user as he will be redirected to `index.php`

Reply:
Fixed.